src/commands/app/logs.js

```diff
@@ -78,7 +78,10 @@ class Logs extends BaseCommand {
 
     try {
       const owConfig = { ow: Object.values(fullConfig.all)[0].ow }
-      await rtLib.printActionLogs(owConfig, this.log, flags.limit, filterActions, flags.strip, flags.poll || flags.tail || flags.watch)
+      const tail = flags.poll || flags.tail || flags.watch
+      const startTime = tail ? Date.now() : 0
+
+      await rtLib.printActionLogs(owConfig, this.log, flags.limit, filterActions, flags.strip, flags.poll || flags.tail || flags.watch, undefined, startTime)
```

Comment:
line 84: `flags.poll || flags.tail || flags.watch` should be just `tail` as you defined in line 81

Reply:
oh, I missed that. 
Thanks!